packages/frames.js/src/getFrameFlattened.ts

```diff
@@ -6,11 +6,43 @@ import type { Frame, FrameFlattened } from "./types";
  * @returns a plain object with frame metadata keys and values according to the frame spec, using their lengthened syntax, e.g. "fc:frame:image"
  */
 export function getFrameFlattened(frame: Frame): FrameFlattened {
+  const openFrames = !!frame.accepts?.length
+    ? {
+        // custom of tags
+        [`of:version`]: frame.version,
```

Comment:
this should be a separate property - right now it's shared between farcaster and open frames and we won't be able to support them diverging

Reply:
isn't that true of all properties? won't we update this if they diverge?

Reply:
hm i guess you're right